homeassistant/components/skybell/config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Config flow for Skybell integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from requests.exceptions import ConnectTimeout, HTTPError
+from skybellpy import Skybell, exceptions
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.typing import ConfigType
+
+from .const import AGENT_IDENTIFIER, DEFAULT_CACHEDB, DOMAIN
+
+
+class SkybellFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Skybell."""
+
+    VERSION = 1
+
+    async def async_step_import(self, user_input: ConfigType) -> FlowResult:
+        """Import a config entry from configuration.yaml."""
+        if self._async_current_entries():
+            return self.async_abort(reason="already_configured")
+        return await self.async_step_user(user_input)
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        errors = {}
+
+        if user_input is not None:
+            email = user_input[CONF_EMAIL]
```

Comment:
does this need to be normalized?
https://developers.home-assistant.io/docs/config_entries_config_flow_handler/#sometimes-acceptable-sources-for-a-unique-id-for-cloud-services

Reply:
Email address can be changed easily by the user so I would not like to see this as the unique id. Should this still be normalized with `lower()`?